grep-0018-modernize-c++.md

```diff
@@ -0,0 +1,417 @@
+# GREP 0018 -- Modernize C++
+
+- Original Author: Thomas Habets <thomas@habets.se>
+- Champion: Thomas Habets <thomas@habets.se>
+- Status: Draft
+
+History:
+- 20-Jan-2020: Initial Draft
+
+## Abstract
+
+As of early 2020 there is lots of pre-C++11 code in GNU Radio, and
+some other pattern of non-idiomatic C++.
```

Comment:
Even if it's obvious, can you add a sentence here that it is our desire to move away from pre-C++11 code, in particular where it incurs Boost dependencies.

Reply:
Basically, what is this GREP proposing? Answer: Remove all the old crap (use better language please :smile: ).

Reply:
I put that under motivation. Do you want me to move it, or copy it?

Reply:
The abstract should be a summary of the entire GREP, so it would be nice if the motivation for this GREP is also in here.

Reply:
Copied.